docs/start/setup-channel.mdx

```diff
@@ -165,8 +165,73 @@ print(result)
     from qiskit_ibm_runtime import QiskitRuntimeService
 
     service = QiskitRuntimeService()
+  ```
+
+## Access IBM Quantum Systems using REST API
+
+Alternatively, you can also access IBM Quantum Systems using REST APIs, enabling you to work with quantum systems from any programming language or framework. 
+
+1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account) and save it to a file named `token`.
+
+2. Optionally obtain a temporary access token by supplying the API token obtained above. This is especially useful if you would like control over tokens, such as token invalidation. Alternatively, you can work directly with your IBM Quantum Platform API token.
```

Comment:
I think it would be easier to show the code for just using the token directly, rather than saving it in a separate file, then we can match the format of the other sections, using `<MY_IBM_QUANTUM_TOKEN>` where the token would be

Reply:
Yeah, I agree.